cmd/buildah/config.go

```diff
@@ -221,14 +221,15 @@ func updateConfig(builder *buildah.Builder, c *cobra.Command, iopts configResult
 	if c.Flag("volume").Changed {
 		if volSpec := iopts.volume; len(volSpec) > 0 {
 			for _, volVal := range volSpec {
-				_, err := os.Stat(volVal) //check if volume exists on disk
-				if err == nil {
-					builder.AddVolume(volVal)
-				} else if strings.HasSuffix(volVal, "-") {
+				if strings.HasSuffix(volVal, "-") {
 					rmVol := strings.TrimSuffix(volVal, "-")
-					builder.RemoveVolume(rmVol)
+					if builder.CheckVolume(rmVol) {
+						builder.RemoveVolume(rmVol)
+					} else {
+						builder.AddVolume(volVal)
```

Comment:
Should we really add the volume at this point?  The user has specified a volume with a dash meaning they want to remove it.  Shouldn't this be a no-op or perhaps a logrus.Debugf saying the volume didn't exist?

Reply:
#1670, https://github.com/containers/buildah/pull/1670#issuecomment-507631089

Checking if the file exists on disk broke buildah, so @rhatdan said to just leave it like this.

Reply:
@TomSweeneyRedHat If we don't add the volume at this point, it was pointed out to us there would be no way to add a volume ending in a "-"

Reply:
OK, think I got it now.  so a volume named 'myvol-' is valid?  Ugh, that makes me like this overall change even more.

My big issue with this logic then is if the user meant to remove 'myvol', but instead typed 'myvoll-'.  Instead of removing 'myvol' as intended, they now still have 'myvol' but also now have 'myvoll' and it all happens silently right?

Should we add an output line here telling the user that we're adding the volume to at least hopefully let them know?

Reply:
I'm okay with that. @rhatdan ?

Reply:
Yes lets put out an info message, telling them myvol does not exist, adding myvol-
Tests are blowing up.